README.md

```diff
@@ -17,3 +17,81 @@ ConformalTracking is free software: you can redistribute it and/or modify it und
 This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License long with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+
+## ConformalTrackingV2
+
+The **ConformalTrackingV2** processor is fully configurable via the steering
+file. The parameter *Steps* takes a "Config" like String.  For example:
+
+    [VXDBarrel]
+    @Collections = VXDTrackerHits
+    @Parameters = MaxCellAngle = 0.035; MaxCellAngleRZ = 0.035;  Chi2Cut = 300;  MinClustersOnTrack = 4;  MaxDistance = 0.015;
+    @Flags = HighPTFit, VertexToTracker
+    @Functions = CombineCollections, BuildNewTracks
+    [VXDEncap]
+    @Collections = VXDEndcapTrackerHits
+    @Parameters = MaxCellAngle = 0.035; MaxCellAngleRZ = 0.035;  Chi2Cut = 300;  MinClustersOnTrack = 4;  MaxDistance = 0.015;
+    @Flags = HighPTFit, VertexToTracker
+    @Functions = CombineCollections, ExtendTracks
+    [TCVC]
+    @Collections = VXDTrackerHits, VXDEndcapTrackerHits
+    @Parameters = MaxCellAngle = 0.035; MaxCellAngleRZ = 0.035;  Chi2Cut = 300;  MinClustersOnTrack = 4;  MaxDistance = 0.015;
+    @Flags = HighPTFit, VertexToTracker
+    @Functions = CombineCollections, BuildNewTracks
+    [LowerCellAngle1]
+    @Collections =
+    @Parameters = MaxCellAngle = 0.175; MaxCellAngleRZ = 0.175;  Chi2Cut = 300;  MinClustersOnTrack = 4;  MaxDistance = 0.015;
+    @Flags = HighPTFit, VertexToTracker, RadialSearch
+    @Functions = BuildNewTracks
+    [LowerCellAngle2]
+    @Collections =
+    @Parameters = MaxCellAngle = 0.35; MaxCellAngleRZ = 0.35;  Chi2Cut = 6000;  MinClustersOnTrack = 4;  MaxDistance = 0.015;
+    @Flags = HighPTFit, VertexToTracker, RadialSearch
+    @Functions = BuildNewTracks
+    [LowerHitNumbers]
+    @Collections =
+    @Parameters = MaxCellAngle = 0.35; MaxCellAngleRZ = 0.35;  Chi2Cut = 6000;  MinClustersOnTrack = 4;  MaxDistance = 0.015;
```

Comment:
In case of not dropping it, the parameters are:
@parameters = MaxCellAngle = 0.05; MaxCellAngleRZ = 0.05; Chi2Cut = 2000; MinClustersOnTrack = 4; MaxDistance = 0.02;
In case of dropping it, make sure SortTracks is done in the previous step (TBD: check whether it is needed at all)